vtex/render-extension-loader
# CHK-192: Add query string to request in order to avoid CORS issue

#### What is the purpose of this pull request?

Add a query string with the store hostname to avoid CORS issue when the browser uses a cached response from another hostname (which has a incorrect `Access-Control-Allow-Origin` header).

#### What problem is this solving?

Workaround to resolve a CORS issue seen in Chrome. The issue is similar to [this chromium bug report](https://bugs.chromium.org/p/chromium/issues/detail?id=260239), but we also added the `Vary: Origin` header as one of the comments suggest to fix the issue and still see the issue happening.

#### How should this be manually tested?

You can see the error by doing the following steps (in Chrome only):

1. Access [Samsung checkout](https://shop.samsung.com/br/checkout) by one of their hosts.
2. Access [Samsung checkout](https://shop.samsung.com.br/checkout) by the other one of their hosts.
3. You can see that the shipping preview and omnishipping (the shipping step in Checkout) won't load, because of a CORS error in the legacy extensions request.

You can now verify that the following snippet of code will work in the console of the second Checkout, because the browser will use a different cache key to store this request's response:

```js
await fetch(
  'https://master--samsungbr.myvtex.com/legacy-extensions/checkout?__disableSSR&locale=pt-BR&v=3&origin=shop.samsung.com.br'
)
```

#### Types of changes
